src/backend/distributed/commands/utility_hook.c

```diff
@@ -890,16 +912,6 @@ static void
 PostStandardProcessUtility(Node *parsetree)
 {
 	DecrementUtilityHookCountersIfNecessary(parsetree);
-
```

Comment:
why do we change the place of this code block?

Reply:
https://github.com/citusdata/citus/pull/4453#discussion_r551944775